server/src/services/role.service.ts

```diff
@@ -0,0 +1,30 @@
+import { RoleRepo } from "../repositories/role.repo.js";
+import ApiError from "../error/api.error.js";
+
+export default class RoleService {
+  async create(role: string) {
+    return await RoleRepo.createRole(role);
```

Comment:
I'd set roleRepo as an private attribute. It's a common practice. Usually data convertors are used in this way